### ISSUE TITLE
Too many "Move Buffer Into..." options?

### ISSUE THREAD

**Issue by macintacos** [VSpaceCode/VSpaceCode#220]
It seems like the `SPC b {h/j/k/l}` mappings are the same as the `SPC b {H/J/K/L}` mappings. Is that what they are in Spacemacs, and that's why they were brought over to this extension? Personally I prefer if a given keymap "area" has one and only one way of doing something, but maybe I'm overlooking something.

If this is intentional, I'll just remap the `HJKL` versions to something more useful for me and close this out.

**Comment by stevenguh** [comment]
The `SPC b {H/J/K/L}` are the same as Spacemacs. We added `SPC b {h/j/k/l}` because the capital keys are quite hard to enter for this common operation. We also didn't have anything map to `{h/j/k/l}` at the time.

Wondering is there something you want to add buffer?

**Comment by macintacos** [comment]
Heyo! Sorry for the delay here.

I'm completely in agreement with you about switching these common operations over to lowercase. As for the uppercase version, I'm testing out the various "duplicate editor X direction" commands to see if I fine them to be useful (I don't really have a strong preference for anything right now):

```json
    {
      "command": "workbench.action.duplicateActiveEditorGroupLeft",
      "icon": "split-horizontal",
      "keys": "b.H",
      "name": "Duplicate buffer to new window left",
      "type": "command"
    },
    {
      "command": "workbench.action.duplicateActiveEditorGroupDown",
      "icon": "split-vertical",
      "keys": "b.J",
      "name": "Duplicate buffer to new window down",
      "type": "command"
    },
    {
      "command": "workbench.action.duplicateActiveEditorGroupUp",
      "icon": "split-vertical",
      "keys": "b.K",
      "name": "Duplicate buffer to new window up",
      "type": "command"
    },
    {
      "command": "workbench.action.duplicateActiveEditorGroupRight",
      "icon": "split-horizontal",
      "keys": "b.L",
      "name": "Duplicate buffer to new window right",
      "type": "command"
    },
```

**Comment by stevenguh** [comment]
This is similar to `SPC w /` or `SPC w -` but have a finer control. I am okie either way, @MarcoIeni any thoughts?

**Comment by MarcoIeni** [comment]
I noticed that `spc b hjkl` and `spc b HJKL` are totally different in spacemacs now in the `develop` branch. I don't think their key bindings make a lot of sense for us, so we are free to do whatever we want :)
I tried your proposal and those key bindings look super good to me! Feel free to submit a PR :)

**Comment by stevenguh** [comment]
Just tried those commands out, they are not what I expected. They duplicate the whole editor group and not just the current buffer, so I am not sure if it makes sense to be in the buffer menu.

**Comment by MarcoIeni** [comment]
Yes, in theory they belong to the windows menu, but if there isn't something better 🤷

**Comment by stevenguh** [comment]
True, either way works me

**Comment by macintacos** [comment]
Sorry about that. I actually don't use VSCode with tabs visible so I didn't realize it duplicated the whole group!

**Comment by macintacos** [comment]
I don't necessarily think that VSpaceCode needs to use this, but I implemented an extension for the aforementioned issue: https://marketplace.visualstudio.com/items?itemName=excessive-dev.buffers

Personally, I will be using this in place of the current value for `SPC b b` 🙂
